style.css

```diff
@@ -0,0 +1,212 @@
+body {
+	margin: 0;
+	font-family:'Montserrat-Regular';
+	font-style: normal;
+	box-sizing: border-box;
+	background: #E5E5E5;
```

Comment:
Codacy found an issue: [Expected "#E5E5E5" to be "#e5e5e5" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Flack_project/pullRequest?prid=5633355)